### PR TITLE
Issue #7145: aligned javadoc/xdoc for CyclomaticComplexity

### PR DESCRIPTION
**Pull request by pbludov**
Issue #7145 (part of #5750)

No changes in the code.

### CODE REVIEW

File: src/xdocs/config_metrics.xml [hunk 1]
```diff
@@ -683,33 +683,38 @@ public class Foo {
...
+        <p>
+          When it comes to code quality measurement by this metric
             level 10 is very good level as a ultimate target (that is hard to archive).
             Do not be ashamed to have complexity level 15 or even higher,
             but keep it below 20 to catch really bad designed code automatically.
-            <br/>Please use Suppression to avoid violations on cases that could not be split in few
+        </p>
+        <p>
+          Please use Suppression to avoid violations on cases that could not be split in few
             methods without damaging readability of code or encapsulation.
```
Comment (rnveach) [inline src/xdocs/config_metrics.xml]: Indent Please to match `methods`.

File: src/xdocs/config_metrics.xml [hunk 2]
```diff
@@ -683,33 +683,38 @@ public class Foo {
...
-          <br/>By pure theory level 1-4 is considered easy to test, 5-7 OK, 8-10
+        </p>
+        <p>
+          By pure theory level 1-4 is considered easy to test, 5-7 OK, 8-10
           consider re-factoring to ease testing, and 11+ re-factor now as testing will be painful.
-
-            <br/>When it comes to code quality measurement by this metric
+        </p>
+        <p>
+          When it comes to code quality measurement by this metric
```
Comment (rnveach) [inline src/xdocs/config_metrics.xml]: same.

File: src/xdocs/config_metrics.xml (hunk 1 above)
Comment (pbludov) [reply src/xdocs/config_metrics.xml]: done

File: src/xdocs/config_metrics.xml (hunk 2 above)
Comment (pbludov) [reply src/xdocs/config_metrics.xml]: done

**Pull request merged**